packages/drivers/odsp-driver/src/odspDelayLoadedDeltaStream.ts

```diff
@@ -190,6 +199,11 @@ export class OdspDelayLoadedDeltaStream {
 					this.currentConnection = undefined;
 				});
 				this.currentConnection = connection;
+				if (websocketEndpoint.sensitivityLabelsInfo !== undefined) {
+					this.emitDeltaConnectionUpdateEvent({
```

Comment:
I think this does nothing, as users had no chance to subscribe to this event, right?
This is that issue with exposing connection objection only after we connect.

Reply:
I've added comment on potential fix a bit below, in emitDeltaConnectionUpdateEvent()

Reply:
Removed it since no longer needed after emitting event on service.